cmd/registry-experimental/cmd/bleve/bleve_test.go

```diff
@@ -181,3 +192,86 @@ func TestSearch(t *testing.T) {
 		}
 	})
 }
+
+func TestServer(t *testing.T) {
+	ctx := context.Background()
+	blevePath := filepath.Join(t.TempDir(), "registry.bleve")
+	port := "8891"
+
+	buildTestRegistry(ctx, t)
+
+	// Start the server.
+	go func() {
+		cmd := Command()
+		cmd.SetArgs([]string{"serve", "--bleve", blevePath, "--port", port})
+		if err := cmd.Execute(); err != nil {
+			log.Fatalf("Execute() with args %+v returned error: %s", cmd.Args, err)
+		}
+	}()
+
+	// Wait for the server to start.
+	_, err := net.DialTimeout("tcp", "localhost:"+port, 2*time.Second)
```

Comment:
@theganyo here's the update to use net.DialTimeout() - thanks, this is much better!